docker-compose.yml

```diff
@@ -24,10 +24,16 @@ services:
       - hmpps
     container_name: localstack-api
     ports:
-      - "4566:4566"
+      - "4666:4666"
```

Comment:
not sure this is correct? usually your mapping from an internal port to an external port?

Reply:
Followed the localstack docs for this?

https://docs.localstack.cloud/references/external-ports/#:~:text=If%20you%20wish%20to%20run,mapped%20to%20non%2Doverlapping%20ranges.

Reply:
could you just do something like 

> 8080:80 | Map port 8080 on the Docker host to TCP port 80 in the container.

So you could do `4666:4566` and that would make this containers internal 4556 port available as 4666?

Reply:
ok, guess localstack is a bit more complicated